version.php

```diff
@@ -16,6 +16,8 @@
 $v_patch = '1';
 $v_tag   = '-dev'; // minor revision number, should be empty for production releases
 
+//This is the CAPEDS with WMT version
+
```

Comment:
I forgot to remove those lines when submitting to the community.  This change was for a customer that has a very specialized custom version of OpenEMR